Steam.lua

```diff
@@ -129,35 +74,50 @@ function main(DEFAULT_STEAM_PATH,USER_ID)
 		local LIBRARY_FOLDERS_VDF_PATH = DEFAULT_STEAM_PATH.."steamapps\\libraryfolders.vdf"
 		local libVDFFile = io.open(LIBRARY_FOLDERS_VDF_PATH,"r")				--open libraryfolders.vdf
 		local dirs = {DEFAULT_STEAM_PATH}										--initialize dirs table with default steam directory preinserted
+		--local lines = {}
 
 		--if vdf file exists
 		if libVDFFile then
 
 			--iterate through all the lines
 			for line in libVDFFile:lines() do
-				local dirline = string.match(line,'^%s*"%d+"%s*"(.*)"')			--match format for only lines with directories
+				local dirline = string.match(line,'%u%:\\\\%a+%d?')			--match format for only lines with directories

```

Comment:
This doesn't seem to match paths more than 1 directory deep (e.g. `D:\Games\Steam`). Instead, I'd suggest:
```suggestion
				local dirline = string.match(line,'"(%u:.+\\.+)"')			--match format for only lines with directories
```
This way, we greedily match from the drive letter to the last backslash, then include the last segment of the path (since that isn't terminated with a backslash).